### ISSUE TITLE
Warning: Operand of null-aware operation '!' has type 'String' which excludes null

### ISSUE THREAD

**Issue by hjd007wwkd** [Baseflow/flutter_cached_network_image#575]
```
../flutter/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-3.0.0-nullsafety.1/lib/src/storage/cache_info_repositories/cache_object_provider.dart:208:29: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
    final oldDbPath = join((await getDatabasesPath())!, '$databaseName.db');
                            ^

```

Flutter gives me this warning message after I upgrade to version 3.0.0-nullsafety

**Comment by vanlooverenkoen** [comment]
This is not related to flutter_cached_network_image but it is an issue in flutter_cache_manager -> https://github.com/Baseflow/flutter_cache_manager/issues/299

**Comment by renefloor** [comment]
Fixed in 3.0.0-nullsafety.2 of flutter_cache_manager, so you can just run pub upgrade
